Sources/WebSocketKit/Exports.swift

```diff
@@ -8,3 +8,5 @@
 @_exported import struct NIOHTTP1.HTTPHeaders
 
 @_exported import struct Foundation.URL
+
+@_exported import Logging
```

Comment:
We shouldn't add this as it makes the whole Logging module part of our public API (I know we've done this for other stuff but that was a mistake 😆 )

```suggestion
@_exported import struct Foundation.URL
```

Reply:
Agree with you. Reverted the change